src/index.html

```diff
@@ -16,17 +16,42 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
+  <body class="page">
+    <!-- <input
       type="text"
       data-qa="keypress"
       placeholder="Try “Los Angeles“"
-    />
+    /> -->
 
-    <input
+    <!-- <input
       type="text"
       data-qa="hover"
       placeholder="Try “Los Angeles“"
-    />
+    /> -->
+    <div class="search-container">
+      <form
+        class="search-container__search-bar search-bar--big"
+        data-qa="big"
+      >
+        <input
+          type="text"
+          class="search-bar__input search-bar__input--big"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+
+      <form
+        class="search-container__search-bar search-bar--small"
+        data-qa="small"
+      >
+        <input
+          type="text"
+          class="search-bar__input search-bar__input--small"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
Similar to the previous `input` tag, the attributes here should also start on a new line with 2-space indentation related to the tag.